### ISSUE TITLE
Latest Version

### ISSUE THREAD

**Issue by PaR1aHNYK** [XProger/OpenLara#460]

**Comment by XProger** [comment]
hi, try to open it via right click -> Open.

**Comment by to-ma-s** [comment]
yes, i unfortunately have the same problem. neither right click -> open will work nor running under rosetta.

**Comment by to-ma-s** [comment]
as seen on discord, typing "xattr -cr /Applications/OpenLara.app" in the terminal helped. now it's working!
